models/conversation.go

```diff
@@ -112,6 +112,41 @@ func (db *DB) GetConversation(id int64) (*Conversation, error) {
 	return conversation, nil
 }
 
+// GetUsersConversations returns all conversations of a user
+func (db *DB) GetUsersConversations(userID int64, sort string) ([]Conversation, error) {
+	var queryString strings.Builder
+	fmt.Fprintf(&queryString, "SELECT c.ID, c.Name, c.Description, c.AvatarURL ")
+	fmt.Fprintf(&queryString, "FROM %s AS c JOIN %s AS m ON c.ID = m.ConversationID ", conversationsTable, mappingsTable)
+	fmt.Fprintf(&queryString, "WHERE UserID=? ORDER BY m.ConversationID ")
+	if sort == "desc" {
+		fmt.Fprintf(&queryString, sort)
+	}
+	rows, err := db.Query(queryString.String(), userID)
+
+	if err != nil {
+		log.Printf("err")
+		return nil, err
+	}
+	defer rows.Close()
+
+	// Create list of conversations
+	c := Conversation{}
+	conversations := make([]Conversation, 0)
+	for rows.Next() {
+		err := rows.Scan(&c.ID, &c.Name, &c.Description, &c.AvatarURL)
+		if err != nil {
+			log.Printf("err 2")
+			return nil, err
+		}
+		conversations = append(conversations, c)
+	}
+
+	if rowCount := len(conversations); err == nil {
```

Comment:
I don't think you need to check for an error here since `rowCount := len(conversations)` doesn't ever return an error. You can just print `rowCount` since you know `conversations` will definitely have a length at this point.